docs/developer/available-plugins/adyen.mdx

```diff
@@ -2,6 +2,62 @@
 title: Adyen
 ---
 
+# Adyen
+
```

Comment:
I would consider to move here a section that starts from line 61:
```
The Adyen plugin allows you to process customer transactions via [Adyen](https://www.adyen.com).
The list of available payments methods can be found [here](https://docs.adyen.com/payment-methods)

:::note
Saleor uses Adyen's checkout API in version [64](https://docs.adyen.com/api-explorer/#/CheckoutService/v64/overview).
If you're using [drop-in](https://docs.adyen.com/online-payments/web-drop-in/integrated-before-5-0-0) or
[components](https://docs.adyen.com/online-payments/web-components/integrated-before-5-0-0) integrations make sure that
you're using Adyen's packages in a version lower than 5.0.0.
:::

:::note
To use the Adyen plugin, make sure that you have configured the plugin.
:::
```

(can't make commit suggestion as it is out of the scope of given changes)

Reply:
Good catch

Reply:
@timuric  what about the `notes`?
```
:::note
 Saleor uses Adyen's checkout API in version [64](https://docs.adyen.com/api-explorer/#/CheckoutService/v64/overview).
If you're using [drop-in](https://docs.adyen.com/online-payments/web-drop-in/integrated-before-5-0-0) or
[components](https://docs.adyen.com/online-payments/web-components/integrated-before-5-0-0) integrations make sure that
you're using Adyen's packages in a version lower than 5.0.0.
:::
:::note
To use the Adyen plugin, make sure that you have configured the plugin.
:::
```
Isn't something that also fit to the intro part?